pyuvdata/uvbeam.py

```diff
@@ -1019,12 +1019,46 @@ def __iadd__(self, other):
         self.__add__(other, inplace=True)
         return self
 
+    def _get_beam(self, stokes):
+        """
+        Get the healpix beam map corresponding to the specififed stokes polarization,
```

Comment:
It doesn't look to me like this is specific to HEALPix beams. If it is, you should add a check that errors if this is called on a non-HEALPix beam